contrib/coreos/user-data.example

```diff
@@ -20,19 +20,59 @@ coreos:
   units:
   - name: etcd.service
     command: start
-  - name: docker-tcp.socket
+  - name: flanneld.service
     command: start
-    enable: true
     content: |
       [Unit]
-      Description=Docker Socket for the API
+      Description=Network fabric for containers
+      Documentation=https://github.com/coreos/flannel
+      Requires=early-docker.service
+      After=etcd.service early-docker.service
+      Before=early-docker.target
 
-      [Socket]
-      ListenStream=2375
-      BindIPv6Only=both
-      Service=docker.service
+      [Service]
+      Type=notify
+      Restart=always
+      RestartSec=5
+      Environment="TMPDIR=/var/tmp/"
+      Environment="DOCKER_HOST=unix:///var/run/early-docker.sock"
+      Environment="FLANNEL_VER=0.4.1"
```

Comment:
@kmala maybe this could be [v0.5.0](https://github.com/coreos/flannel/releases/tag/v0.5.0)? 
(It contains a fix for vxlan)